replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/QueryServiceImpl.java

```diff
@@ -52,17 +57,19 @@ public QueryServiceImpl(
    * Provides a stream for retrieving all saved user queries as {@link Query}s.
    *
    * @return all the user queries saved on the system
+   * @throws QueryException if an exception occurs while querying the local catalog
    */
   @Override
   public Stream<Query> queries() {
     Filter filter = builder.attribute(Core.METACARD_TAGS).is().equalTo().text("workspace");
-    QueryImpl query = new QueryImpl(filter);
-    query.setPageSize(100);
+    PropertyName sortProperty = new PropertyNameImpl(Core.METACARD_MODIFIED);
+    SortBy sortPolicy = new SortByImpl(sortProperty, SortOrder.DESCENDING);
+    QueryImpl query = new QueryImpl(filter, 1, 100, sortPolicy, false, 0L);
     QueryResponse queryResponse;
     try {
       queryResponse = framework.query(new QueryRequestImpl(query));
     } catch (UnsupportedQueryException | SourceUnavailableException | FederationException e) {
```

Comment:
❓ Can `UnsupportedQueryException` actually happen in our case?

Reply:
I don't think so. Since I'm querying the local catalog I don't think any of these exceptions will occur.